manim/animation/fading.py

```diff
@@ -1,13 +1,63 @@
-"""Fading in and out of view."""
+"""Fading in and out of view.
+
+.. manim:: Example
+    :hide_source:
+
+    class Example(Scene):
+        def construct(self):
+            s1 = Square().set_color(BLUE)
+            s2 = Square().set_color(BLUE)
+            s3 = Square().set_color(BLUE)
+            s4 = Square().set_color(BLUE)
+            s5 = Square().set_color(BLUE)
+            s6 = Square().set_color(RED)
+            s7 = Square().set_color(RED)
+            s8 = Square().set_color(RED)
```

Comment:
Any ideas of how to make this nicer?
Looking forward to this new config system!

Reply:
`Square(color=BLUE)`?

Reply:
okay, a few characters less, but I think that does not change a lot, nevertheless, thanks for the idea!

Reply:
something like s = [Square(color=c) for c in [BLUE,BLUE,BLUE,BLUE,BLUE,RED,RED,RED]] but i don't think is is more understandable.